lessons/introR/main.Rmd

```diff
@@ -0,0 +1,291 @@
+---
```

Comment:
While this is true that Jekyll also uses YAML, I've used Rmd files in my own blog. I'll check into it though.